deps/wazuh_testing/wazuh_testing/tools/performance/visualization.py

```diff
@@ -87,6 +91,24 @@ def _load_dataframes(self):
                 new_csv = pd.read_csv(df_path, index_col="Timestamp", parse_dates=True)
                 self.dataframe = pd.concat([self.dataframe, new_csv])
 
+    def _unify_dataframes(self):
+        """Unify dataframe values."""
```

Comment:
Improve this docstring. It unify dataframe regarding the Daemon value.

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5500/commits/ee85a1dc18d61450298897d04af637de0c4f2051